tests/components/prometheus/test_init.py

```diff
@@ -139,3 +147,171 @@ async def test_view(prometheus_client):  # pylint: disable=redefined-outer-name
         'entity="sensor.sps30_pm_1um_weight_concentration",'
         'friendly_name="SPS30 PM <1µm Weight concentration"} 3.7069' in body
     )
+
+
+@pytest.fixture(name="mock_client")
+def mock_client_fixture():
+    """Mock the prometheus client."""
+    with mock.patch(f"{PROMETHEUS_PATH}.prometheus_client") as client:
+        counter_client = mock.MagicMock()
+        client.Counter = mock.MagicMock(return_value=counter_client)
+        setattr(counter_client, "labels", mock.MagicMock(return_value=mock.MagicMock()))
+        yield counter_client
+
+
+@pytest.fixture
+def mock_bus(hass):
+    """Mock the event bus listener."""
+    hass.bus.listen = mock.MagicMock()
+
+
+@pytest.mark.usefixtures("mock_bus")
+async def test_minimal_config(hass, mock_client):
+    """Test the minimal config and defaults of component."""
+    config = {prometheus.DOMAIN: {}}
+    assert await async_setup_component(hass, prometheus.DOMAIN, config)
+    await hass.async_block_till_done()
+    assert hass.bus.listen.called
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5745133)